src/main/java/seedu/address/storage/JsonAdaptedAppointment.java

```diff
@@ -58,8 +58,8 @@ public JsonAdaptedAppointment(Appointment source) {
         appointmentType = source.getAppointmentType().value;
         appointmentDateTime = source.getAppointmentDateTime().toString();
         personId = source.getPerson().getPersonId();
-        sickness = source.getSickness().value;
-        medicine = source.getMedicine().value;
+        sickness = source.getSickness().map(sickness -> sickness.value).orElse("");
+        medicine = source.getMedicine().map(sickness -> sickness.value).orElse("");
```

Comment:
Should we leave in the DG somewhere that signifies that "" in the JSON refers to when the sickness and medicine fields are left empty?